cclient/response/get_trytes.c

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) 2018 IOTA Stiftung
+ * https://github.com/iotaledger/entangled
+ *
+ * Refer to the LICENSE file for licensing information
+ */
+
+#include "get_trytes.h"
+
+get_trytes_res_t* get_trytes_res_new() {
+  get_trytes_res_t* hashes = NULL;
+  utarray_new(hashes, &ut_str_icd);
+  return hashes;
+}
+
+void get_trytes_res_add(get_trytes_res_t* trytes_array, const char* trytes) {
```

Comment:
is it being used?

Reply:
No, the array in the response is deserialized via `json_array_to_utarray`, so we might not need this function, same as `get_tips_res_add`, I think it can be removed.